UIforETW/Support.cpp

```diff
@@ -62,6 +62,7 @@ void CUIforETWDlg::TransferSettings(bool saving)
 		{ L"SampledStacks", &bSampledStacks_ },
 		{ L"FastSampling", &bFastSampling_ },
 		{ L"GPUTracing", &bGPUTracing_ },
+        { L"CLRTracing", &bCLRTracing_ },

```

Comment:
It looks like you edited the file using spaces and the rest of it uses tabs, so the lines don't line up when github displays them with an odd tab size. This repeats in quite a few places.